ecosystem/indexer-grpc/indexer-grpc-fullnode/src/runtime.rs

```diff
@@ -38,6 +42,14 @@ pub struct FullnodeDataService {
     pub output_batch_size: u16,
 }
 
+// TODO: move it to data service.
+pub struct RawDataService {
+    pub context: Arc<Context>,
+    pub processor_task_count: u16,
+    pub processor_batch_size: u16,
+    pub output_batch_size: u16,
```

Comment:
These are identical to the fields in FullnodeDataService, can you either just use the same struct or extract the common stuff into another struct?

Reply:
done.